apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -477,6 +477,15 @@ const (
 	// (i.e., fleet-member-<member-name>) on the hub cluster yet.
 	ClusterResourcePlacementSynchronizedConditionType ClusterResourcePlacementConditionType = "ClusterResourcePlacementSynchronized"
 
+	// ClusterResourcePlacementOverriddenConditionType indicates whether all the selected resources have been overridden
+	// successfully before applying to the target cluster if there is any override defined.
+	// Its condition status can be one of the following:
+	// - "True" means all the selected resources are successfully overridden before applying to the target cluster or
+	// override is not needed if there is no override defined.
+	// - "False" means some of them have failed. We will place some detailed failure in the FailedResourcePlacement array.
+	// - "Unknown" means we haven't finished the override yet.
```

Comment:
for a newly added condition, there is also a case of "not set". I assume that the default is "no override needed"?

Reply:
do you mean the resource itself does not have override policy? you want to distinguish it with a separate condition?

In the current api design, if no override needed, the ClusterResourcePlacementOverridden condition will be set as true.